src/App.js

```diff
@@ -0,0 +1,464 @@
+				Ride: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Nature: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Diplomacy: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Engineering: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Preform: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Disable Device: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Geography: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Escape Artist: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+				Slight of Hand: 
+                	<input type="submit" value="+"/>
+		</div>
+		<div>
+			<p>Languages:</p>
+			<p>Languages that are avilable</p>
+		</div>
+		<div>
+			<p>Feats:</p>
+			<p>Feats that are avilable</p>
+		</div>
+		<div>
+			<p>Special Abilities:</p>
+			<p>Special Abilities that are avilable</p>
+		</div>
+            </TabPanel>
+            <TabPanel>
+		<div>
+			<p>Cantrips:</p>
+			<p>Cantrips that are avilable</p>
+		</div>
+		<div>
+			<p>Prepared Spells:</p>
+			<p>Prepared Spells that are avilable</p>
+		</div>
+		<div>
+			<p>Active Spell Effects Abilities:</p>
+			<p>Active Spell Effects that are avilable</p>
+		</div>
+		<h2>Spell Slots</h2>
+		<div>
+			Level	Max	Current
+		</div>
+		<div>
+			Level 1:
+		</div>
+		<div>
+			Level 2:
+		</div>
+		<div>
+			Level 3:
+		</div>
+		<div>
+			Level 4:
+		</div>
+		<div>
+			Level 5:
+		</div>
+		<div>
+			Level 6:
+		</div>
+		<div>
+			Level 7:
+		</div>
+		<div>
+			Level 8:
+		</div>
+		<div>
+			Level 9:
+		</div>
+		<div>
+			<p>Known Spells:</p>
+			<p>Known Spells that are avilable</p>
+		</div>
+            </TabPanel>
+
+            <TabPanel>
+                <h3>Currency</h3>
+		<div>
+          		Platinum:
+          		<input type="text" name="platinum"/>
+		</div>
+		<div>
+          		Gold:
+          		<input type="text" name="gold"/>
+		</div>
+		<div>
+          		Silver:
+          		<input type="text" name="silver"/>
+		</div>
+		<div>
+          		Copper:
+          		<input type="text" name="copper"/>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+                	<input type="submit" value="+"/>
+				<p>Equitment</p>
+				<p>Equitment that is avilable</p>
+		</div>
+		<div>
+                	<input type="submit" value="-"/>
+                	<input type="submit" value="+"/>
+				<p>Bags</p>
```

Comment:
I think that scrollable column element should be part of this pull request and used in multiple places, including here.